keps/sig-storage/2268-non-graceful-shutdown/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: non graceful shutdown
+kep-number: 2264
+authors:
+  - "@yastij"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+status: implemented
```

Comment:
KEP status should be `implementable`. Only after graduation, KEP status becomes `implemented`